local/postman/puris-integration-test.postman_collection.json

```diff
@@ -0,0 +1,1299 @@
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{CUSTOMER_PURIS_BACKEND}}/catena/stockView/material-stocks",
+									"host": [
+										"{{CUSTOMER_PURIS_BACKEND}}"
+									],
+									"path": [
+										"catena",
+										"stockView",
+										"material-stocks"
+									]
+								}
+							},
+							"response": []
+						}
+					],
+					"auth": {
+						"type": "apikey",
+						"apikey": [
+							{
+								"key": "value",
+								"value": "{{CUSTOMER_PURIS_BACKEND_API_KEY}}",
+								"type": "string"
+							},
+							{
+								"key": "key",
+								"value": "X-API-KEY",
+								"type": "string"
+							}
+						]
+					},
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									""
+								]
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									""
+								]
+							}
+						}
+					]
+				},
+				{
+					"name": "SupplierInit",
+					"item": [
+						{
+							"name": "Create Semiconductor Product",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"response is ok\", function() {",
+											"    pm.response.to.have.status(200);",
+											"})"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n  \"materialFlag\": false,\n  \"productFlag\": true,\n  \"ownMaterialNumber\": \"{{MATERIAL_NUMBER_SUPPLIER}}\",\n  \"materialNumberCx\": \"{{MATERIAL_NUMBER_CX}}\",\n  \"name\": \"Semiconductor\"\n}"
+								},
+								"url": {
+									"raw": "{{SUPPLIER_PURIS_BACKEND}}/catena/materials",
+									"host": [
+										"{{SUPPLIER_PURIS_BACKEND}}"
+									],
+									"path": [
+										"catena",
+										"materials"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Get All Materials",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"response is ok\", function() {",
+											"    pm.response.to.have.status(200);",
+											"})",
+											"",
+											"pm.test(\"Verify response contains one material objects\", function () {",
+											"    pm.expect(pm.response.json()).to.be.an(\"array\").with.lengthOf(1);",
+											"});",
+											"",
+											"pm.test(\"Verify ownMaterialNumber values\", function () {",
+											"    const materialStock = pm.response.json()[0];",
```

Comment:
Variable name "materialStock" might be a bit misleading. 
The catena/materials/all - endpoint will only deliver Dto's of Material Entities. 
This has nothing to do with stocks of any kind.

Reply:
good catch, renamed to material